docs/release_notes/9_5.md

```diff
@@ -52,79 +52,57 @@ In addition to the Azure Marketplace, Rocky Linux is available for free on the A
 
 ## Major changes
 
-For a complete list of major changes, see the [upstream listing here](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/9.4_release_notes/overview#overview-major-changes).
+For a complete list of major changes, see the [upstream listing here](https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/pdf/9.5_release_notes/Red_Hat_Enterprise_Linux-9-9.5_Release_Notes-en-US.pdf#__WKANCHOR_6).
 
 Highlights and new features from this release are detailed next.
 
-### Image builder
-
-* From Rocky Linux 9.4 forward, you can specify arbitrary custom mount points except for specific paths that are reserved for the operating system
-* Creation of different partitioning modes is now available including `auto-lvm`, `lvm`, and `raw`
-* Tailor options for a profile and add them to your blueprint customizations. You can accomplish this by selecting and unselecting options to add and remove rules.
-
 ### Security
 
-Listed below are security-related highlights in the latest Rocky Linux 9.4 release. For a complete list of security-related changes, see the [upstream link here](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/9.4_release_notes/new-features#new-features-security).
+Listed below are security-related highlights in the latest Rocky Linux 9.5 release. For a complete list of security-related changes, see the [upstream link here](https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/pdf/9.5_release_notes/Red_Hat_Enterprise_Linux-9-9.5_Release_Notes-en-US.pdf#%23new-features-security).
```

Comment:
Move to HTML link: https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/9.5_release_notes/new-features#new-features-security